packages/twenty-front/src/modules/object-record/record-picker/single-record-picker/components/SingleRecordPickerMenuItems.tsx

```diff
@@ -93,56 +86,60 @@ export const SingleRecordPickerMenuItems = ({
     singleRecordPickerSelectedIdComponentState,
   );
 
+  const singleRecordPickerShowSkeleton = useRecoilComponentValueV2(
+    singleRecordPickerShowSkeletonComponentState,
+  );
+
+  const singleRecordPickerShowInitialLoading = useRecoilComponentValueV2(
+    singleRecordPickerShowInitialLoadingComponentState,
+  );
+
+  const searchHasNoResults =
+    recordsToSelect.length === 0 && filteredSelectedRecords?.length === 0;
+
   return (
     <SelectableList
       selectableListInstanceId={selectableListComponentInstanceId}
       selectableItemIdArray={selectableItemIds}
       hotkeyScope={DropdownHotkeyScope.Dropdown}
       focusId={focusId}
     >
-      {loading ? (
-        <DropdownMenuSkeletonItem />
+      {emptyLabel && (
+        <SelectableListItem
+          key={'select-none'}
+          itemId={'select-none'}
+          onEnter={() => {
+            setSelectedRecordId(undefined);
+            onRecordSelected();
+          }}
```

Comment:
Where possible it's best to create named functions to avoid un-necessary creation of anonymous functions on re-renders.
```tsx
function onEnter(){
  setSelectedRecordId(undefined);
  onRecordSelected();
}
...
onEnter={onEnter}
```